aips/create_dips_job.py

```diff
@@ -58,29 +53,29 @@ def setup_logger(log_file, log_level="INFO"):
             },
         },
         "loggers": {
-            "create_dip": {"level": log_level, "handlers": ["console", "file"]}
+            "dip_workflow": {"level": log_level, "handlers": ["console", "file"]}
         },
     }
 
     logging.config.dictConfig(CONFIG)
 
 
-def main(
-    ss_url, ss_user, ss_api_key, location_uuid, tmp_dir, output_dir, database_file
-):
-    LOGGER.info("Processing AIPs in SS location: %s", location_uuid)
+def main(args):
```

Comment:
I'm not particularly convinced that `main(args)` is better than what we had before, under the premise of it's best to be explicit, but I understand the motivation. I'd say that having too many parameters is undesirable, but having many and lose visibility it's going to make the problem bigger down the road.

Reply:
I agree, I'll be explicit until we implement a better solution. Thanks!